Remove scaleGracePeriod from Hadoop wordcount sample

**Problem description**
When configuring and building `pravega-samples` with the new release version to come, the Hadoop wordcount example fails with the following exception:
```
Task :hadoop-connector-examples:compileJava 
/home/raul/Documents/workspace/test/testRelease/pravega-samples/hadoop-connector-examples/src/main/java/io/pravega/example/hadoop/wordcount/PravegaOutputFormat.java:103: error: cannot find symbol
                .transactionTimeoutScaleGracePeriod(DEFAULT_TXN_SCALE_GRACE_PERIOD_MS)
                ^
  symbol:   method transactionTimeoutScaleGracePeriod(long)
  location: class EventWriterConfigBuilder
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
1 error
```
The `scaleGracePeriod` has been removed from Pravega transactions and it needs to be removed from this example too.

**Problem location**
Hadoop wordcount sample (`PravegaOutputFormat.java`).

**Suggestions for an improvement**
Remove operations on the obsolete `scaleGracePeriod`.

--- a/gradle.properties
+++ b/gradle.properties
@@ -21,4 +21,4 @@ flinkVersion=1.4.0
 hadoopVersion=2.8.1
 scalaVersion=2.11.8
 sparkVersion=2.2.0
-hadoopConnectorVersion=0.3.0-17.a171316-SNAPSHOT
+hadoopConnectorVersion=0.3.0-20.1a0dde3-SNAPSHOT

--- a/hadoop-connector-examples/src/main/java/io/pravega/example/hadoop/wordcount/PravegaOutputFormat.java
+++ b/hadoop-connector-examples/src/main/java/io/pravega/example/hadoop/wordcount/PravegaOutputFormat.java
@@ -52,7 +52,6 @@ public class PravegaOutputFormat<V> extends OutputFormat<String, V> {
 
     private static final long DEFAULT_TXN_TIMEOUT_MS = 30000L;
     private static final long DEFAULT_TXN_MAX_EXECUTION_TIME_MS = 30000L;
-    private static final long DEFAULT_TXN_SCALE_GRACE_PERIOD_MS = 30000L;
     private static final long DEFAULT_PING_LEASE_MS = 30000L;
 
     // client factory
@@ -100,7 +99,6 @@ public class PravegaOutputFormat<V> extends OutputFormat<String, V> {
 
         EventStreamWriter<V> writer = clientFactory.createEventWriter(streamName, deserializer, EventWriterConfig.builder()
                 .transactionTimeoutTime(DEFAULT_TXN_TIMEOUT_MS)
-                .transactionTimeoutScaleGracePeriod(DEFAULT_TXN_SCALE_GRACE_PERIOD_MS)
                 .build());
 
         return new PravegaOutputRecordWriter<V>(writer);

